checker/check-response-property-one-of-updated_test.go

```diff
@@ -77,5 +87,15 @@ func TestResponsePropertyOneOfRemoved(t *testing.T) {
 			Path:        "/pets",
 			Source:      "../data/checker/response_property_one_of_removed_revision.yaml",
 			OperationId: "listPets",
+		},
+		{
+			Id:          "response-property-one-of-removed",
+			Text:        "removed 'Dark brown types' from the '/oneOf[#/components/schemas/Fox]/breed' response property 'oneOf' list for the response status 200",
```

Comment:
without the change from this PR, this line would be: `removed 'BaseSchema[1]' from the...`

Reply:
I agree that adding the schema title makes it easier to read the result, however, I am concerned that the output becomes ambiguous.
Previously, the user could tell whether the change was inline (like BaseSchema[1]) or a ref otherwise, after this PR this information is lost.
Perhaps, we should use: 'BaseSchema[1]: Dark brown types'

What do you think?

Reply:
made the change. Indeed, having only the title might be confusing (as that's not a class name), thanks for the suggestion.